gulpfile.js

```diff
@@ -59,7 +67,25 @@ gulp.task('browserify_watch', function () {
     .pipe(gulp.dest(paths.buildRoot));
 });
 
+//sass
+gulp.task('styles', function() {
+  return gulp.src(paths.sassFiles)
+  .pipe(sourcemaps.init())
+  .pipe(sass())
+  .pipe(concating('style.css'))
+  .pipe(minifycss())
+  .pipe(gulp.dest(paths.buildProdStyles))
```

Comment:
writing files into prod folder all the time while developing, slows build system.
Find a way to optimize this - write only necessary files depending on a chosen type of build.